returnn/util/basic.py

```diff
@@ -123,10 +123,9 @@ def is_64bit_platform():
 class BackendEngine:
   """
   Stores which backend engine we use in RETURNN.
-  E.g. Theano or TensorFlow.
+  Currently only TensorFlow.
```

Comment:
```suggestion
  E.g. TensorFlow or PyTorch.
```

Reply:
As I said, this is WIP, do not look at those details yet. When I made this commit we did not have PyTorch yet.

Reply:
I don't exactly understand the complaint. Sure it can only help or not?

Reply:
Ok if it does not bother you then sure, you can list all that already.